templates/fungible-asset-template/move/sources/launchpad.move

```diff
@@ -93,21 +100,32 @@ module launchpad_addr::launchpad {
         });
         move_to(sender, Config {
             admin_addr: signer::address_of(sender),
+            pending_admin_addr: option::none(),
             mint_fee_collector_addr: signer::address_of(sender),
         });
     }
 
     // ================================= Entry Functions ================================= //
 
-    public entry fun update_admin(sender: &signer, new_admin: address) acquires Config {
-        assert!(is_admin(signer::address_of(sender)), E_NOT_ADMIN);
+    public entry fun set_pending_admin(sender: &signer, new_admin: address) acquires Config {
```

Comment:
could we doc on what these functions are for?

Reply:
added docs on all public functions